.github/workflows/ci_bench_block_execution.yaml

```diff
@@ -61,11 +61,11 @@ jobs:
         with:
           tool: hyperfine@1.16
 
-      - name: Fetch base binary
+      - name: Fetch main binary
```

Comment:
but why would you want to compare against main? You want to compare against base actually.

Reply:
We had a PR where we had some changes that were not being accurately portrayed since the base branch was not main, so we changed it to always compare with main (it's merged, but this is a fix for it).

Let's discuss if we want to compare either base or main.